### ISSUE TITLE
Feature Request: Localization for different tax jurisdictions

### ISSUE THREAD

**Issue by jimkoen** [bigcapitalhq/bigcapital#123]
I hope I speak for more people than just myself when I say that in regards to OSS accounting software, the Number 1 thing keeping me from adoption is the fact, that most if not all are focused on US specifics.

I'm reasonably sure that I would not be able to use this software for my bookkeeping in Germany for example,
because Germany has specific quasi-standards on how to submit financial records to it's tax authorities.

I'm not asking for the developers to support specifics for all tax jurisdictions on the planet, but providing
guidelines on how to port bigcapitals functionality to other jurisdictions would be a game changer.

**Comment by sammcj** [comment]
Yeah would be nice to have an alternative to Xero etc... here in Australia and New Zealand.

**Comment by abouolia** [comment]
That thing absolutely is in our mind, the next big thing that we still working on is the tax compliance, and wanted to make it customizable to fit any need, I can't deny the software is still new and right now all our concentration is to stabilize the product and fill the missed gaps.

**Comment by abhibeckert** [comment]
Tax in Australia is pretty straightforward. Here's how it works:

 - buy a widget for $30, you pay $3 sales tax on top of the price
 - sell the widget for $40, you charge an extra $4 sales tax on top

Once a month (less often for small businesses), you subtract the amount you paid ($3) from the amount you charged ($4) and pay $1 to the tax office along with a simple online form (where you basically say "I paid $3, I collected $4, here is $1).

That's it for most businesses. One federal tax system nationwide for (nearly) everyone. Some products like staple foods are tax exempt, but then you just charge no sales tax.

Things get a little more complex when it comes to employee payroll tax, but it's still nothing like the USA.

**Comment by abouolia** [comment]
@abhibeckert Taxes feature has released in the last version, it has everything to achieve what you're talking about.

**Comment by curiousgeorgios** [comment]
hi @abouolia just been testing out some of the stuff in bigcapital as like @abhibeckert mentioned the potential for a Xero replacement is huge in Australia. I know bigcapital is still in early stages, but users should have the option to apply tax to all transactions. At the moment I don't think there's an option to specify a tax for Expenses, which is important. Thanks for all the work you guys are doing! :)
